File: examples/notResource/notResource.py
import json
import iam_floyd as statement

s = (
    # doc-start
    statement.S3() \
        .allow() \
        .not_resources() \
        .to_delete_bucket() \
        .on_bucket('example-bucket')
    # doc-end
).to_json()

pretty = json.dumps(s, indent=4, sort_keys=True)
print(pretty)

File: examples/notResource/notResource.ts
import { deploy, out } from '../../helper/typescript/typescript_test';
import * as statement from '../../lib';

function getStatement() {
  return (
    // doc-start
    new statement.S3()
      .allow()
      .notResource()
      .toDeleteBucket()
      .onBucket('example-bucket')
    // doc-end
  );
}
const s = [getStatement()];
out(s);
deploy(s);
